packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1827,7 +1855,7 @@ export default class NewVm extends BaseComponent {
               </Item>
             </SectionContent>
           ),
-          template && template.virtualizationMode === 'hvm' && (
+          this._isHvm() && (
```

Comment:
Better define a `const`.

Reply:
Why a const would be better than a computed ?

Reply:
I don't mean to replace the computed but to call the computed one time.